weed/shell/command_volume_check_disk.go

```diff
@@ -7,16 +7,22 @@ import (
 	"fmt"
 	"github.com/seaweedfs/seaweedfs/weed/operation"
 	"github.com/seaweedfs/seaweedfs/weed/pb"
+	"github.com/seaweedfs/seaweedfs/weed/pb/master_pb"
 	"github.com/seaweedfs/seaweedfs/weed/pb/volume_server_pb"
 	"github.com/seaweedfs/seaweedfs/weed/storage/needle_map"
 	"golang.org/x/exp/slices"
 	"google.golang.org/grpc"
 	"io"
 	"math"
 	"net/http"
+	"sync"
 	"time"
 )
 
+const (
+	VolumePulseSeconds = 5
```

Comment:
Not a good place for this volume server setting.

Reply:
make new place